shanoir-ng-front/src/app/datasets/shared/dataset.service.ts

```diff
@@ -139,7 +139,9 @@ export class DatasetService extends EntityService<Dataset> {
             AppUtils.BACKEND_API_DATASET_URL + '/downloadStatistics', { observe: 'response', responseType: 'blob', params: params})
             .toPromise().then(
             response => {
-                this.downloadIntoBrowser(response);
+                if (response.status != 204) {
+                    this.consoleService.log('error', 'Error during creation of statistics.');
```

Comment:
On last little change, could you display a console message (Or directly on the page after the 4 textbox in green) a message stating that the tats export has started, and to check the "jobs" panel when it's a success ?